modules/nw-dns-forward.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * dns/dns-operator.adoc
+
+[id="nw-dns-forward_{context}"]
+= Using DNS forwarding
+
+You can use DNS forwarding to override the forwarding configuration identified in `etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone.
+
+.Procedure
+
+. Modify the DNS Operator object named `default`:
++
+----
+$ oc edit dns.operator/default
+----
+.. This allows the Operator to create and update the ConfigMap named `dns-default` with additional server configuration blocks based on `Server`. If none of the servers has a zone that matches the query, then name resolution falls back to the name servers that are specified in `/etc/resolv.conf`.
++
+.Sample DNS
+[source,YAML]
```

Comment:
Not sure if it renders the same capitalized, but suggest `source,yaml` to be consistent throughout our doc set.